.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: [3.8]
-        toxenv: [py38, integration, quality, translations]
+        toxenv: [py38, django32, django42, integration, quality, translations]
```

Comment:
`py38` is redundant. It would make sense to use both Django 3.2 and 4.2 for `integration`, `quality`, and `translations`, though.